src/translate/compute.rs

```diff
@@ -26,16 +26,19 @@ pub fn translate_computation(meta: &TranslateMetadata, operation: ArithOp, left:
     match meta.arith_module {
         ArithType::BcSed => {
             let (left, right) = (left.unwrap_or_default(), right.unwrap_or_default());
-            let mut math_lib_flag = true;
-            // Removes trailing zeros from the expression
-            let sed_regex = "/\\./ s/\\.\\{0,1\\}0\\{1,\\}$//";
+            let mut scale = "";
+            let mut after_bc = "";
+            let sed_regex = "| sed \"/\\./ s/\\.\\{0,1\\}0\\{1,\\}$//\"";
```

Comment:
This `sed` replacement is suspect in two ways: (i) it makes two separate regex calls, one in the initial test for a single ".", and another in the actual substitution; and (ii) it fails to strip trailing "0"s from "123.000".  I would like to propose an alternative, which also IMO has the additional advantage of being slightly more readable:
```
"| sed \"s/\\(\\.\\(0\\|[0-9]*[1-9]\\)\\)00*$/\\1/\""
```
This finds a "." followed by either a single "0", or any digits followed by a non "0" digit, and replaces that capture with the capture itself, discarding any subsequent "0"s.

Reply:
I also suggest ensuring that whatever unit tests we have for this include the following cases (that is, if it's possible to make `bc` output the ones with extra "0"s):
```
0
123
1230
12300
123000
123.0
123.00
123.000
123.45
123.450
123.4500
123.45000
123.0045
123.00450
123.004500
123.0045000
```
Should output:
```
0
123
1230
12300
123000
123.0
123.0
123.0
123.45
123.45
123.45
123.45
123.0045
123.0045
123.0045
123.0045
```

Reply:
the test right now is that now https://github.com/amber-lang/amber/blob/master/src/tests/validity/div.ab if you can suggest me the calculation to do I will add there

Reply:
I agree with @hdwalters

Reply:
I am crap with math, if you can help me to write the tests to generate numbers like this on a division I can add them.